packages/perseus/src/widgets/interactive-graphs/graph-locked-layer.tsx

```diff
@@ -4,14 +4,16 @@ import * as React from "react";
 import LockedLine from "./locked-line";
 import LockedPoint from "./locked-point";
 
-import type {LockedFigure} from "../../perseus-types";
+import type {LockedFigure, Range} from "../../perseus-types";
 
 type Props = {
     lockedFigures: ReadonlyArray<LockedFigure>;
+    range: [Range, Range];
```

Comment:
minor: this type is a bit confusing. "The range is a pair of `Range`s". I might prefer to use the `Interval` type from Mafs, and label the elements of the pair:

```ts
import type {Interval} from "mafs";
// ...
type Props = {
  // ...
  range: [x: Interval, y: Interval];
}
```